### DISCUSSION TITLE
Simpler bash completion

### DISCUSSION THREAD

**Discussion by lyderic** [casey/just#1504] [Ideas]
To add _just_ completion to bash, the easiest is to add the following line in ~/.bashrc:

```
source <(just --completions bash)
```

No need to fiddle with completions scripts in /etc/bash_completion.d or whatever, which can vary according to linux distribution and might need to be updated at every new _just_ version.

To be complete, these are the lines I use to also have completion for the 'j' alias:

```
alias j=just
source <(just --completions bash)
complete -F _just -o bashdefault -o default j
```

Maybe this is worth adding to the documentation in:
https://just.systems/man/en/chapter_63.html

Please note I have tested this only on bash, my bash version is: 
```GNU bash, version 5.1.16(1)-release (x86_64-pc-linux-gnu)```

**Comment by casey** [comment]
Yeah, good point, this should definitely be in the readme.

**Comment by Steiniche** [comment]
The docs could also state that 

`eval "$(just --completions bash)"`

can be added to `.bashrc` and auto complete will work in a bash environment.
